config/environments/development.rb

```diff
@@ -58,5 +58,8 @@
   # Allow disabling the webpack dev server from the settings
   config.webpack.dev_server.enabled = SETTINGS.fetch(:webpack_dev_server, true)
   config.webpack.dev_server.https = SETTINGS.fetch(:webpack_dev_server_https, false)
-  config.hosts << SETTINGS[:fqdn]
```

Comment:
Would it be useful to continue ensure this in every dev setup by default? We can't really rely on `settings.yaml` in every setup (like katello-devel doesn't contain it: https://github.com/theforeman/puppet-katello_devel/blob/master/templates/settings.yaml.erb).

Reply:
> Would it be useful to continue ensure this in every dev setup by default? We can't really rely on `settings.yaml` in every setup

Yes, I think that makes more sense. It also removes the need for special treatment of `'FQDN'`.